### PR TITLE
docs: error response when external auth is enabled

### PR DESCRIPTION
**Pull request by mrnyax**
In this case, the external authentication system is used to manage password change

### CODE REVIEW

**Review by hjohannsen — APPROVED** [review]
Looks good to me.

File: apiary.apib [hunk 1]
```diff
@@ -3137,12 +3137,26 @@ For Core Platform versions 2021.05 to 2022.02, the following apply:
...
         {
-        "links": {},
-            "error": {
-                "detail": "The user does not have the required privileges to perform the operation.",
-                "type": "insufficientPrivileges",
-                "title": "Insufficient privileges",
-                "status": 403
+            "links": {},
+                "error": {
+                    "detail": "The user does not have the required privileges to perform the operation.",
```
Comment (madeline-brennan) [inline apiary.apib:3142]:
```suggestion
                    "detail": "The user doesn't have the privileges required to perform the operation.",
```

File: apiary.apib [hunk 2]
```diff
@@ -3137,12 +3137,26 @@ For Core Platform versions 2021.05 to 2022.02, the following apply:
...
+        }
+
++ Response 403 (application/json)
+
+        // when external authentication is enable you should not be able to change password through the API
+        {
+            "links": {},
+                "error": {
+                    "reference": "82a781cb-059a-4039-a17d-af8e20bec019",
+                    "detail": "The user is not allowed to change password.",
```
Comment (madeline-brennan) [inline apiary.apib:3156]:
```suggestion
                    "detail": "The user isn't allowed to change the password.",
```

File: apiary.apib [hunk 3]
```diff
@@ -3137,12 +3137,26 @@ For Core Platform versions 2021.05 to 2022.02, the following apply:
...
+                    "detail": "The user does not have the required privileges to perform the operation.",
+                    "type": "insufficientPrivileges",
+                    "title": "Insufficient privileges",
+                    "status": 403
+            }
+        }
+
++ Response 403 (application/json)
+
+        // when external authentication is enable you should not be able to change password through the API
```
Comment (madeline-brennan) [inline apiary.apib:3151]:
```suggestion
        // when external authentication is enabled, you shouldn't be able to change password through the API
```

**Review by madeline-brennan — CHANGES_REQUESTED** [review]
UX writer review

**Review by hjohannsen — APPROVED** [review]

**Review by madeline-brennan — APPROVED** [review]

**Review by madeline-brennan — APPROVED** [review]

**Pull request merged**